test/inputs/anonymous_functions.js

```diff
@@ -7,5 +7,9 @@ var testObj = {
 };
 
 testObj.somemethod('I shall not pass');
-testObj.gettext("I'm gonna get translated, yay!");
+testObj.pgettext("I'm gonna get translated, yay!");
 testObj.ngettext("I'm also gonna get translated!", "I'm the plural form!", 2);
+testObj.pgettext("context1", "I am translated in context!");
```

Comment:
That looks like a type. It was correct before
`testObj.gettext("I'm gonna get translated, yay!")`